backend/src/controller.ts

```diff
@@ -284,4 +290,58 @@ export class Controller {
 
     return data
   }
+
+async getTransferTimes(params: any): Promise<TimeToBridgeStats> {
+  const { sourceChainSlug, destinationChainSlug } = params
+
+  if (!sourceChainSlug || !destinationChainSlug) {
+    return
```

Comment:
suggestion: throw an error if chains not specified

```suggestion
if (!sourceChainSlug) {
  throw new Error('sourceChainSlug is required')
}
if (!destinationChainSlug) {
  throw new Error('destinationChainSlug is required')
}